tests/test_e2e_10_mef_eline.py

```diff
@@ -1369,3 +1369,121 @@ def test_155_removing_evc_metadata_persistent(self):
         data = response.json()
         assert my_key not in data['metadata']
         assert len(data['metadata']) > 0
+
+    def test_160_create_untagged_evc(self):
```

Comment:
Aldo, please let's also include docstrings summarizing what the test will actually test, its name is clear and already gives the idea, but we're also maintaining docstrings for all tests

Reply:
Changed